WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
+  serialize :combined_round_cutoff, CombinedRoundCutoff
+  serialize :next_round_cutoff, Cutoff
```

Comment:
@jonatanklosko If I knew what the terms mean, I could probably give some name suggestions. I tried reading the regulations, but found them unreadable.